src/main/java/com/dev/bankingservice/entity/Role.java

```diff
@@ -0,0 +1,28 @@
+package com.dev.bankingservice.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "role")
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(EnumType.STRING)
+    @Column(name = "role_name")
+    private RoleName roleName;
+
+    public enum RoleName {
+        CUSTOMER,
+        MANAGER
```

Comment:
RoleName should be ADMIN and USER by requirements

Reply:
Thank you, renamed the role to `ADMIN` and `USER`.